### ISSUE TITLE
Problems using keyboard shortcuts

### ISSUE THREAD

**Issue by perberos** [perberos/Mate-Desktop-Environment#55]
some keys are not working well.
I do not have idea when this start to happens.
I has not added any new code on the sourcecode.

**Comment by perberos** [comment]
On microsoft wired keyboard 600 - linuxmint:

```
<zorba> 7~7~7~7~7~7~
<zorba> that gets typed when i do ctrl+alt+del
<zorba> 7~
```

**Comment by iarspider** [comment]
Dell XPS M1530: multimedia keys not working, but properly recognized by "gnome keyboard shortcuts"

**Comment by perberos** [comment]
This was fixed on 2011.12.08. Thankyou.

https://github.com/mate-desktop/mate-settings-daemon/commit/eb986a271cb90e22c42c5a789bf414987a3f22b1
https://github.com/mate-desktop/mate-control-center/commit/38701ae3a78144705a70eaf0628449efbaad7076
